src/Compilers/CSharp/Portable/Symbols/Symbol.cs

```diff
@@ -600,6 +600,22 @@ public override bool Equals(object obj)
         }
 
         public bool Equals(ISymbol other)
+        {
+            return this.Equals(other as Symbol, SymbolEqualityComparer.Default.CompareKind);
+        }
+
+        public bool Equals(ISymbol other, SymbolEqualityComparer equalityComparer)
```

Comment:
>public bool Equals(ISymbol other, SymbolEqualityComparer equalityComparer) [](start = 8, length = 74)

Are there any callers to this overload? If not, please remove.

Reply:
Yes, this is the new public interface.

---
In reply to: [305994568](https://github.com/dotnet/roslyn/pull/37247#discussion_r305994568) [](ancestors = 305994568)